controlplane/src/core/repositories/OrganizationRepository.ts

```diff
@@ -963,21 +963,23 @@ export class OrganizationRepository {
     const features = await this.getFeatures({ organizationId: input.organizationID });
 
     // fallback limits @todo: think of a better way to make this dynamic
-    const limits: Record<string, number> = {
+    const limits: Limits = {
       'analytics-retention': 7,
       'tracing-retention': 7,
       'changelog-retention': 7,
       'breaking-change-retention': 7,
       'trace-sampling-rate': 0.1,
+      'federated-graphs': 1,
+      users: 1,
```

Comment:
I specifically didn't add federated-graphs and users here, so it's not limited by default for on prem

Reply:
I thought limits aren't enforced when billing is disabled. Does it mean we can set very high limits here because in production we have to ensure that plans are set?

Reply:
That would probably be better indeed, i'm not 100% happy with how this is setup as you can read in the comment, but i re-used what was already there.

Reply:
ok